docs/blog/2020-06-23-Reconfiguring-Gatsby-Days/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Gatsby Days Reconfigured"
+date: 2020-06-23
+author: Dustin Schau
+excerpt: With all the momentous events of recent weeks, we decided -- after a lot of discussion and thought -- to reconfigure this month’s planned Gatsby Days. Now is not the right time to take attention and space away from the Black Lives Matter movement. Transforming Gatsby Days from a live event into a video and blog content series allows us to share all the amazing speakers and learning opportunities of the live event in a way that the community can access as the time is right for each of us. Today’s post is the first in the series, presenting all the product and program announcements that we had planned for the original online event.
+tags:
+  - announcements
+  - gatsby-days
+  - community
+---
+
+Events over this past month have been truly momentous. While many of us struggle to recognize and address the damage caused by systemic racial injustice and oppression in the US, at Gatsby we are doing our best to meaningfully support the Black Lives Matter movement. This is why we made the decision to reconfigure Gatsby Days from the live online event originally planned for today and tomorrow.
+
+We feel now is simply not the time to take attention and space away from the Black Lives Matter movement. Transforming Gatsby Days from an event into a video and blog content series allows us to share all the amazing speakers and learning opportunities of the live event in a way that the community can access as the time is right for each of us.
+
+Today’s blog post is going to be a long one, because this is where we are sharing the new programs and products we had planned to announce at this week’s Gatsby Days. We are following up with individual posts featuring recorded presentations from the Gatsby community members who were scheduled to speak. We are sorry to miss gathering, even virtually, and we thank everyone for understanding -- and supporting -- the change. Until we can meet again, please enjoy the recorded presentations and demos from Gatsby Days Reconfigured.
+
+**[New from Gatsby](#New-from-Gatsby)**
+
+- [Gatsby Partner Program](#Gatsby-Partner-Program)
```

Comment:
These need to be lowercased.

Reply:
You can grab the hashes by navigating to the blog post, and then hovering over a header and clicking, e.g.

<img width="503" alt="Screen Shot 2020-06-23 at 8 58 33 AM" src="https://user-images.githubusercontent.com/3924690/85426755-bdc33a00-b52f-11ea-993b-510c4b58d9a5.png">

Reply:
Thanks Dustin!